src/commands/geodist.json

```diff
@@ -1,7 +1,7 @@
 {
     "GEODIST": {
         "summary": "Returns the distance between two members of a geospatial index.",
-        "complexity": "O(log(N))",
```

Comment:
that's really odd.. how on earth did it happen that this command got O(log(N))?
@itamarhaber?